.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: Community
-    url: https://community.gitpod.io
-    about: Please ask and answer usage questions here.
+    url: https://www.gitpod.io/chat
```

Comment:
Delete this entire file so that the files within gitpod-io/.github becomes the single source of truth? Adjust gitpod-io/.github if required.

Reply:
Hey @ghuntley, I think that will work well for `contact_links`.For this repo, I think we want to specifically avoid blank issues. For example, so we can collect meaningful info, like the workspace name having trouble.

Do we have any repos where we want to collect `blank issues`, or do we want all users to use the templates in all Gitpod repos? Personally I like the idea of using templates across the repos. :wink:

Reply:
@ghuntley let's plan on continuing to use these repo level template files for the short term (that way we can get folks to Discord). Long term, I'll create a new issue - I'm not sure how best to use Github org level templates (to both consolidate templates, but also override the org level ones, when needed).

Reply:
Opened #7415